lib/rules/no-document-location.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+
+const { docUrl } = require('../util/doc-url');
+
+module.exports = {
+    meta: {
+        docs: {
+            description: 'disallow document.location use, fixable to window.location.',
+            category: 'Locker',
+            url: docUrl('no-document-location'),
+        },
+        schema: [],
+    },
+
+    create(context) {
+        return {
+            CallExpression(node) {
```

Comment:
As far I understand the rule, accessing `document.location` should fail. In this case the `MemberExpression` visitor should be used instead of `CallExpression`.

Reply:
Ah yep, copypasta error. Fixing :)